docs/draft/rest_api.md

```diff
@@ -0,0 +1,34 @@
+# First Draft for the REST API
+> This is only the first draft. Some adaptations are still possible.
+## Data that needs to be provided by the API [GET]
+- current status of the transformer
+  ```
+  GET status/
```

Comment:
It would be quite useful to be able to query all currently running Transformations. Maybe the status request should return those.